debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SnapshotIT.java

```diff
@@ -254,6 +256,65 @@ public void takeSchemaOnlySnapshotAndSendHeartbeat() throws Exception {
         Assertions.assertThat(record.topic()).startsWith("__debezium-heartbeat");
     }
 
+    @Test
+    public void blacklistColumn() throws Exception {
+        connection.execute(
+                "CREATE TABLE blacklist_column_table_a (id int, name varchar(30), amount integer primary key(id))",
+                "CREATE TABLE blacklist_column_table_b (id int, name varchar(30), amount integer primary key(id))"
+        );
+        connection.execute("INSERT INTO blacklist_column_table_a VALUES(10, 'some_name', 120)");
+        connection.execute("INSERT INTO blacklist_column_table_b VALUES(11, 'some_name', 447)");
+        TestHelper.enableTableCdc(connection, "blacklist_column_table_a");
+        TestHelper.enableTableCdc(connection, "blacklist_column_table_b");
+
+        final Configuration config = TestHelper.defaultConfig()
+                .with(SqlServerConnectorConfig.SNAPSHOT_MODE, SnapshotMode.INITIAL)
+                .with(SqlServerConnectorConfig.COLUMN_BLACKLIST, "testDB.dbo.blacklist_column_table_a.amount")
```

Comment:
I will try to shorten the fully-qualified column name, namely, database is redundant. It should be schema.table.column.

Reply:
Yes, indeed.

Reply:
This is hard. TableId with database name included as well is used all around the code :/ I can't just pass `null` and catalog.

I can't just prepend `databaseName.` to blacklisting patterns either.

Reply:
Ok, so this should be fixed now (DBZ-1089), i.e. the test needs adapting.

Reply:
done